adapters/Unity/UnityAdapter/GADMAdapterUnityBannerAd.m

```diff
@@ -13,104 +13,103 @@
+    NSString *_placementID;
 }
 
-- (nonnull instancetype)initWithGADMAdNetworkConnector:(nonnull id<GADMAdNetworkConnector>)connector
-                                               adapter:(nonnull id<GADMAdNetworkAdapter>)adapter {
-  self = [super init];
-  if (self) {
-    _adapter = adapter;
-    _connector = connector;
-  }
-  return self;
+- (instancetype)initWithGADMAdNetworkConnector:(nonnull id<GADMAdNetworkConnector>)connector
+                                       adapter:(nonnull id<GADMAdNetworkAdapter>)adapter {
+    if (![UnityAds isInitialized]) {
+        return nil;
+    }
+    self = [super init];
+    if (self) {
+        _adapter = adapter;
+        _connector = connector;
+    }
+    return self;
 }
 
 - (void)loadBannerWithSize:(GADAdSize)adSize {
-  id<GADMAdNetworkConnector> strongConnector = _connector;
-  id<GADMAdNetworkAdapter> strongAdapter = _adapter;
-
-  if (!strongConnector || !strongAdapter) {
-    NSLog(@"Adapter Error: No GADMAdNetworkConnector nor GADMAdNetworkAdapter found.");
-    return;
-  }
-
-  if (![UnityAds isInitialized]) {
-    NSString *gameID = [strongConnector.credentials[kGADMAdapterUnityGameID] copy];
-    [GADMAdapterUnitySingleton.sharedInstance initializeWithGameID:gameID];
-  }
-
-  _placementID = [strongConnector.credentials[kGADMAdapterUnityPlacementID] copy];
-  _bannerAd = [[UADSBannerView alloc] initWithPlacementId:_placementID size:adSize.size];
-
-  if (!_bannerAd) {
-    NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(
-        GADMAdapterUnityErrorAdObjectNil, @"Unity banner failed to initialize.");
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
-  }
-
-  _bannerAd.delegate = self;
-  [_bannerAd load];
+    id<GADMAdNetworkConnector> strongConnector = _connector;
+    id<GADMAdNetworkAdapter> strongAdapter = _adapter;
+    
+    if (!strongConnector || !strongAdapter) {
+        NSLog(@"Adapter Error: No GADMAdNetworkConnector nor GADMAdNetworkAdapter found.");
+        return;
+    }
+    _placementID = [strongConnector.credentials[kGADMAdapterUnityPlacementID] copy];
+    if (!_placementID) {
+        NSError *error = GADUnityErrorWithDescription(kMISSING_ID_ERROR);
+        [strongConnector adapter:strongAdapter didFailAd:error];
+        return;
+    }
+    _bannerAd = [[UADSBannerView alloc] initWithPlacementId:_placementID size:adSize.size];
+    if (!_bannerAd) {
+        NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(GADMAdapterUnityErrorAdObjectNil, @"Unity banner failed to initialize.");
+        [strongConnector adapter:strongAdapter didFailAd:error];
+        return;
+    }
+    [UnityAds setDebugMode:YES];
+    _bannerAd.delegate = self;
+    [_bannerAd load];
 }
 
 - (void)stopBeingDelegate {
-  _bannerAd = nil;
-  _bannerAd.delegate = nil;
+    _bannerAd = nil;
+    _bannerAd.delegate = nil;
 }
 
 #pragma mark UADSBannerView Delegate methods
 
 - (void)bannerViewDidLoad:(UADSBannerView *)bannerView {
```

Comment:
I added logging statements to all of these methods, I think it could be helpful for debugging